src/org/labkey/test/params/experiment/SampleSetDefinition.java

```diff
@@ -170,4 +172,16 @@ protected String getKind()
         }
         return options;
     }
+
+    @Override
+    protected @NotNull String getSchemaName()
```

Comment:
Odd place for the @NotNull annotation.

Reply:
That's IntelliJ's template for overriding a method. I hadn't notice that before. I guess it makes some kind of sense to have `NotNull` close to the method's return type.

Reply:
FWIW, that's my preferred location for ```@NotNull``` on the return type